1-exercises/A-undefined/exercise.js

```diff
@@ -28,9 +29,9 @@ function sayHelloToUser(user) {
     console.log(`Hello ${user}`);
 }
 
-sayHelloToUser();
+sayHelloToUser("Omer"); // for this example we miss to add value when we call the function.
 
 
 // Example 4
 let arr = [1,2,3];
-console.log(arr[3]);
+console.log(arr[3]);// because the arr has 3 item and the firest item start with 0 and 1 and 2 but there is no for item.
```

Comment:
I think you mean similar to me,  //in this example we don't have arr[3], because we just have [0],[1].[2]and we don't have it.